web_external/Datasets/datasetPage.pug

```diff
@@ -32,9 +32,6 @@
       tr
         td.isic-listing-item-table-label Owner
         td= dataset.get('owner')
-      tr
-        td.isic-listing-item-table-label Signature
-        td= dataset.get('signature')
   if dataset.canAdmin()
```

Comment:
Do these permission checks need to be updated? It looks like being a Dataset Contributor is insufficient to access the metadata endpoints.

Reply:
Fixed.